provider/linear-docs/auth.ts

```diff
@@ -0,0 +1,111 @@
+// Identical to provider/linear-issues/auth.ts.
```

Comment:
If this is identical, should we just created a shared lib file for them, or import one from the other?

Reply:
I thought about it but decided to keep it simple for now with code duplication. I do not expect this code (local auth scripts) to be changed. WDYT?

Reply:
It's a fair whack of code… but we don't have CI that runs across both if you just intend on changing one… so probably not a bad idea to keep separate.